### PR TITLE
DE translation brightness_level

### PR DESCRIPTION
**Pull request by mymyke**
Added brightness_level for DE language
Now one should be able to set brightnes to min and max in german. 

Added Translation for turquoise and Pink

### CODE REVIEW

File: sentences/de/light_HassLightSet.yaml [hunk 1]
```diff
@@ -24,3 +24,12 @@ intents:
           - "<setzen> die Farbe [<lichter>|<alle_lichter>] [<area>] auf {color}"
           - "die Farbe [<lichter>|<alle_lichter>] [<area>] auf {color} <setzen>"
         response: color_area
+
+        # Max/Min brightness
+      - sentences:
+          - "<setzen> [[[die] Helligkeit [von|von der|von den|der] ]| [die|das]] <name> auf [die|das] {brightness_level:brightness}  [ein]"
```
Comment (mib1185) [inline sentences/de/light_HassLightSet.yaml:30]: we can use the `<von_dem>` expansion rule here, further the `<name>` rule does already include articels

File: sentences/de/_common.yaml [hunk 2]
```diff
@@ -28,11 +28,21 @@ lists:
...
+      - in: "türkis"
+        out: "turquoise"
   brightness:
     range:
       type: "percentage"
       from: 0
       to: 100
+  brightness_level:
+    values:
+      - in: (max|[maximale|höchste|volle] (Helligkeit|Stufe) |maximum)
```
Comment (mib1185) [inline sentences/de/_common.yaml:42]:
```suggestion
      - in: (max|[maximale|höchste|volle] (Helligkeit|Stufe)|maximum)
```

File: sentences/de/_common.yaml [hunk 3]
```diff
@@ -28,11 +28,21 @@ lists:
...
+        out: 100
+      - in: (minimum|minimale|niedrigste)
```
Comment (mib1185) [inline sentences/de/_common.yaml:44]: why not also use `Helligkeit|Stufe` here?

File: tests/de/light_HassLightSet.yaml
Comment (mib1185) [inline tests/de/light_HassLightSet.yaml:1]: after the changes above, please add a test sentences for every possible variant, also for the "set minimum" sentences

**Review by mib1185 — CHANGES_REQUESTED** [review]
Hi @mymyke 
thanks for this contribution 👍 
There are some comments from my side, further the tests are failing, please fix this

File: sentences/de/_common.yaml [hunk 4]
```diff
@@ -28,11 +28,21 @@ lists:
         out: "purple"
       - in: "braun"
         out: "brown"
+      - in: "rosa"
+        out: "pink"
+      - in: "türkis"
+        out: "turquoise"
```
[the remark this answers is not in the document]
Comment (mymyke) [reply sentences/de/_common.yaml]: I removed that

File: tests/de/light_HassLightSet.yaml
Comment (mymyke) [reply tests/de/light_HassLightSet.yaml:1]: Tests were added now

File: sentences/de/light_HassLightSet.yaml [hunk 5]
```diff
@@ -24,3 +24,12 @@ intents:
           - "<setzen> die Farbe [<lichter>|<alle_lichter>] [<area>] auf {color}"
           - "die Farbe [<lichter>|<alle_lichter>] [<area>] auf {color} <setzen>"
         response: color_area
+
+        # Max/Min brightness
+      - sentences:
+          - "<setzen> [[die] Helligkeit [von|<von_dem>]] <name> auf [die|das] {brightness_level:brightness}  [ein]"
+          - "[die] Helligkeit [von|<von_dem>] <name> auf [die|das] {brightness_level:brightness} <setzen>"
```
Comment (mib1185) [inline sentences/de/light_HassLightSet.yaml:31]:
```suggestion
          - "<setzen> [[die] Helligkeit [<von_dem>]] <name> auf [die|das] {brightness_level:brightness}  [ein]"
          - "[die] Helligkeit [<von_dem>] <name> auf [die|das] {brightness_level:brightness} <setzen>"
```

the expansion rule `<von_dem>` already covers "von" , "von dem" and "vom"

File: sentences/de/_common.yaml [hunk 6]
```diff
@@ -33,6 +33,12 @@ lists:
       type: "percentage"
       from: 0
       to: 100
+  brightness_level:
+    values:
+      - in: (max|(maximale|höchste|volle) (Helligkeit|Stufe)|maximum)
```
Comment (mib1185) [inline sentences/de/_common.yaml:38]:
```suggestion
      - in: (max[(imale|imum)]|höchste|volle)
```
Sorry, but after some thought I'm think we should remove the `(Helligkeit|Stufe)` here and adjust the sentence definitions accordingly to create meaningful sentences. the same counts for the minimum values below.

**Review by mib1185 — CHANGES_REQUESTED** [review]
Hi @maddox 
sorry for late review, but busy days 🙁 
After some thought about this PR, I've added some new comments

File: tests/de/light_HassLightSet.yaml [hunk 7]
```diff
@@ -1,5 +1,42 @@
...
+      - Setze die Schlafzimmerlampe auf maximum
+      - Stelle die Helligkeit von der Schlafzimmerlampe auf max
+      - Ändere die Helligkeit von der Schlafzimmerlampe auf die maximale Helligkeit
+      - Stelle die Helligkeit von der Schlafzimmerlampe auf die maximale Helligkeit
+      - Setze die Helligkeit der Schlafzimmerlampe auf die maximale Stufe
+      - Veränder die Helligkeit von der Schlafzimmerlampe auf die höchste Stufe
+      - Stelle die Schlafzimmerlampe auf das Maximum ein
+      - Schlafzimmerlampe auf die höchste Helligkeit setzen
+      - die Helligkeit der Schlafzimmerlampe auf das Maximum stellen.
+      - das Schlafzimmerlampe auf volle Helligkeit ändern
```
Comment (mib1185) [inline tests/de/light_HassLightSet.yaml:14]:
```suggestion
      - Stelle die Schlafzimmerlampe auf maximum
      - Stelle die Schlafzimmerlampe auf das maximum
      - Stelle die Schlafzimmerlampe auf die höchste Helligkeit
      - Stelle die Schlafzimmerlampe auf die höchste Stufe ein
      - Stelle die Schlafzimmerlampe auf das maximum ein
      - Stelle die Schlafzimmerlampe auf höchste Helligkeit
      - Stelle die Schlafzimmerlampe auf höchste Stufe ein
      - Stelle die Schlafzimmerlampe auf maximum ein
      - Stelle Helligkeit der Schlafzimmerlampe auf maximum
      - Stelle die Helligkeit der Schlafzimmerlampe auf maximum
      - Stelle die Helligkeit der Schlafzimmerlampe auf das maximum
      - Stelle die Helligkeit der Schlafzimmerlampe auf die höchste Stufe
      - Stelle die Helligkeit der Schlafzimmerlampe auf die höchste Stufe ein
      - Stelle die Helligkeit der Schlafzimmerlampe auf das maximum ein
      - Stelle die Helligkeit der Schlafzimmerlampe auf maximum
      - Stelle die Helligkeit der Schlafzimmerlampe auf höchste Stufe
      - Stelle die Helligkeit der Schlafzimmerlampe auf höchste Stufe ein
      - Stelle die Helligkeit der Schlafzimmerlampe auf maximum ein
      - Stelle Helligkeit von der Schlafzimmerlampe auf die höchste Stufe ein
      - Stelle Helligkeit von der Schlafzimmerlampe auf maximale Stufe ein
      - Helligkeit der Schlafzimmerlampe auf maximum stellen
      - die Helligkeit der Schlafzimmerlampe auf maximum stellen
      - die Helligkeit der Schlafzimmerlampe auf das maximum stellen
      - die Helligkeit der Schlafzimmerlampe auf die höchste Stufe stellen
      - die Helligkeit von der Schlafzimmerlampe auf maximum stellen
      - die Helligkeit von der Schlafzimmerlampe auf höchste Stufe stellen
      - die Helligkeit von der Schlafzimmerlampe auf die höchste Stufe stellen
      - Ändere die Helligkeit von der Schlafzimmerlampe auf die maximale Helligkeit
      - Veränder die Helligkeit von der Schlafzimmerlampe auf die höchste Stufe
      - das Schlafzimmerlampe auf volle Helligkeit ändern
```

File: tests/de/light_HassLightSet.yaml [hunk 8]
```diff
@@ -1,5 +1,42 @@
...
+  - sentences:
+      - Setze die Schlafzimmerlampe auf minimum
+      - Stelle die Helligkeit von der Schlafzimmerlampe auf min
+      - Ändere die Helligkeit von der Schlafzimmerlampe auf die minimale Helligkeit
+      - Stelle die Helligkeit von der Schlafzimmerlampe auf die minimale Helligkeit
+      - Setze die Helligkeit der Schlafzimmerlampe auf die minimale Stufe
+      - Veränder die Helligkeit von der Schlafzimmerlampe auf die niedrigste Stufe
+      - Stelle die Schlafzimmerlampe auf das Minimum ein
+      - Schlafzimmerlampe auf die niedrigste Helligkeit setzen
+      - die Helligkeit der Schlafzimmerlampe auf das Minimum stellen.
```
Comment (mib1185) [inline tests/de/light_HassLightSet.yaml:32]:
```suggestion
      - Stelle die Schlafzimmerlampe auf minimum
      - Stelle die Schlafzimmerlampe auf das minimum
      - Stelle die Schlafzimmerlampe auf die niedrigste Helligkeit
      - Stelle die Schlafzimmerlampe auf die niedrigste Stufe ein
      - Stelle die Schlafzimmerlampe auf das minimum ein
      - Stelle die Schlafzimmerlampe auf niedrigste Helligkeit
      - Stelle die Schlafzimmerlampe auf niedrigste Stufe ein
      - Stelle die Schlafzimmerlampe auf minimum ein
      - Stelle Helligkeit der Schlafzimmerlampe auf minimum
      - Stelle die Helligkeit der Schlafzimmerlampe auf minimum
      - Stelle die Helligkeit der Schlafzimmerlampe auf das minimum
      - Stelle die Helligkeit der Schlafzimmerlampe auf die niedrigste Stufe
      - Stelle die Helligkeit der Schlafzimmerlampe auf die niedrigste Stufe ein
      - Stelle die Helligkeit der Schlafzimmerlampe auf das minimum ein
      - Stelle die Helligkeit der Schlafzimmerlampe auf minimum
      - Stelle die Helligkeit der Schlafzimmerlampe auf niedrigste Stufe
      - Stelle die Helligkeit der Schlafzimmerlampe auf niedrigste Stufe ein
      - Stelle die Helligkeit der Schlafzimmerlampe auf minimum ein
      - Stelle Helligkeit von der Schlafzimmerlampe auf die niedrigste Stufe ein
      - Stelle Helligkeit von der Schlafzimmerlampe auf niedrigste Stufe ein
      - Helligkeit der Schlafzimmerlampe auf minimum stellen
      - die Helligkeit der Schlafzimmerlampe auf minimum stellen
      - die Helligkeit der Schlafzimmerlampe auf das minimum stellen
      - die Helligkeit der Schlafzimmerlampe auf die niedrigste Stufe stellen
      - die Helligkeit von der Schlafzimmerlampe auf minimum stellen
      - die Helligkeit von der Schlafzimmerlampe auf niedrigste Stufe stellen
      - die Helligkeit von der Schlafzimmerlampe auf die niedrigste Stufe stellen
      - Ändere die Helligkeit von der Schlafzimmerlampe auf die niedrigste Helligkeit
      - Veränder die Helligkeit von der Schlafzimmerlampe auf die niedrigste Stufe
      - das Schlafzimmerlampe auf niedrigste Helligkeit ändern
```

**Review by mib1185 — APPROVED** [review]
LGTM, thx @mymyke 👍
